build/repo.targets

```diff
@@ -1,9 +1,13 @@
 <Project>
   <ItemGroup>
     <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1" />
+    <Scenarios Include="-n Plaintext -o KestrelHttpServer@dev --kestrelThreadCount 1 --kestrelThreadPoolDispatch false" />
```

Comment:
Yes, these two features are mostly unrelated.  KestrelThreadCount controls the number of libuv threads, while KestrelThreadPoolDispatch controls whether app code is run on the libuv thread or a threadpool thread.